db/views/latest_blood_pressures_per_patient_per_months_v01.sql

```diff
@@ -0,0 +1,15 @@
+SELECT DISTINCT ON (patient_id, year, month)
+    blood_pressures.id AS bp_id,
+    patient_id,
```

Comment:
For ease of reading through this file, can we include the appropriate table prefixes on _all_ fields, instead of just the ambiguous ones?